patches/minecraft/net/minecraft/tileentity/TileEntityChest.java.patch

```diff
@@ -28,6 +29,7 @@
 +        return super.getCapability(capability, facing);
 +    }
 +
++    @Nullable
```

Comment:
Can this one ever actually return null?

Reply:
Looking at it more closely, it certainly seems it's not supposed to.
I'll remove the annotation here.